### DISCUSSION TITLE
Support for ffmpeg 6.1 ?

### DISCUSSION THREAD

**Discussion by chinbhat** [bluenviron/mediamtx#2837] [Questions] [answered]
### Question

Hello,

When does MTX plan to support ffmpeg 6.1 ? 

Looking to publish HEVC RTMP using hooks and i see that it's supported only in ffmpeg6.1. The current binary in the docker:latest is ffmpeg 6.0

I tried using statistically compiled binary, but its not able to resolve hostnames. 

Appreciate any help.

**Comment by aler9** [comment, marked as answer]
Hello, the Docker image is based on Linux Alpine 3.18, i've bumped it to 3.19, that contains FFmpeg 6.1. This will be available  in the next release.

In the meanwhile, you can either build a custom Docker image that includes FFmpeg and MediaMTX (there are tons of instructions anywhere) or use the standalone executable in a system in which FFmpeg is already present.

**Reply by chinbhat** [reply]
That’s wonderful. Thank you so much. 

I’m the meanwhile that’s what I did. Manually added ffmpeg 6.1 to the container. 

It works wonderful for an E2E HEVC streaming. 

Thanks for an amazing work on the product.

[a comment by an automated account is not shown]
